### ISSUE TITLE
4.3 Two Types

### ISSUE THREAD

**Issue by mcr** [ietf-rats-wg/architecture#187]
````
   An Attester consists of at least one Attesting Environment and at
   least one Target Environment.  In some implementations, the Attesting

````

Can you say what these are?  I don’t think they need to be in the glossary, but some clue as to how you tell one from another might help see why there’s a distinction

**Comment by dthaler** [comment]
My opinion is that the 2nd paragraph of the section (the paragraph after the sentence quoted) already defines them.  I don't know how to make that clearer.

**Comment by mcr** [comment]
fixed by #269.
